src/vs/workbench/contrib/positronPlots/browser/components/webviewPlotInstance.tsx

```diff
@@ -32,17 +32,22 @@ export const WebviewPlotInstance = (props: WebviewPlotInstanceProps) => {
 		// Only claim if the plot is visible to avoid rendering the webview when
 		// the parent view pane is collapsed.
 		if (props.visible) {
-			client.claim(this);
+			client.activate().then(() => {
+				client.claim(this);
+			});
 		}
 		return () => {
 			client.release(this);
 		};
 	}, [props.plotClient, props.visible]);
 
 	useEffect(() => {
-		if (webviewRef.current) {
-			props.plotClient.layoutWebviewOverElement(webviewRef.current);
-		}
+		const client = props.plotClient;
+		client.activate().then(() => {
+			if (webviewRef.current) {
+				client.layoutWebviewOverElement(webviewRef.current);
+			}
+		});
```

Comment:
Since this `useEffect()` doesn't have a dependency array it gets called a lot and it seems like it's calling `client.activate()` in quick enough succession that the `this._webview.value = await this.createWebview();` line doesn't finish before the second call. 
When I put a break point at `NotebookOutputPlotClient.createWebview()` it gets hit twice on first display for an altair plot. 

```python
import altair as alt
import pandas as pd

source = pd.DataFrame({
    'a': ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I'],
    'b': [28, 55, 43, 91, 81, 53, 19, 87, 52]
})

alt.Chart(source).mark_bar().encode(
    x='a',
    y='b'
)
``` 
It doesn't seem to be harming anything now but I wonder if we should debounce somewhere to prevent potential problems from occuring.

Reply:
Ah great point, made the same mistake in the last PR!